start-client/src/components/common/explore/ExploreModal.js

```diff
@@ -15,16 +15,21 @@ class ExploreModal extends React.Component {
   constructor(props) {
     super(props)
     this.ref = React.createRef()
+    this.state = {
+      button: 'Copy',
+    }
   }
 
   onCopy = () => {
-    toast.success('Your file has been copied in the clipboard with success.')
+    this.setState({ button: 'Copied!' })
```

Comment:
Is `Copied!` the message that the user is going to see? Don't we want something more neutral?

Reply:
Nevermind, I thought the message on the guides was a bit different.